packages/client/hmi-client/src/services/beaker.ts

```diff
@@ -0,0 +1,11 @@
+import axios, { AxiosHeaders } from 'axios';
+
+const BeakerAPI = axios.create({
+	baseURL: '/beaker',
+	headers: new AxiosHeaders()
+});
+
+export const summarizeNotebook = async (notebook: any) => {
+	const { data } = await BeakerAPI.post('/summary', { notebook });
```

Comment:
I am not familiar with this endpoint, does it accept just `any` and tries its best no matter what?
If it has a loose type can you add a comment?
If it has a strict type can you type it?

Reply:
FYI this is not on our end:

https://github.com/jataware/beaker-kernel/blob/886b2b3913ca1460f0301a5cd97cbcf15de609bc/beaker_kernel/server/main.py#L217-L221

Reply:
I'll add a comment on that. It's from Jataware's beaker repo https://github.com/jataware/beaker-kernel/blob/886b2b3913ca1460f0301a5cd97cbcf15de609bc/beaker-ts/src/notebook.ts#L405